subprojects/docs/src/docs/release/notes.md

```diff
@@ -175,6 +175,16 @@ Disable options are now sorted after their enable option, for example the option
 
 See the [task options](userguide/custom_tasks.html#sec:listing_task_options) user manual section for more information.
 
+### Dry run mode for test execution
+
+It is possible now to run a test without actual test execution. 
+
+This can be useful for quick verifying of tests filtering configuration.
```

Comment:
```suggestion
This can be useful for quickly verifying which tests will actually run when [filtering a test suite](userguide/java_testing.html#test_filtering).
```